Project.toml

```diff
@@ -31,7 +31,7 @@ FASTX = "1.1, 2"
 GLM = "1.1"
 LinearAlgebra = "1"
 NLopt = "0.5.1, 0.6, 1"
-PhyloNetworks = "1"
+PhyloNetworks = "1.1"
```

Comment:
This seems to be needed because `check_valid_gammas` was only implemented in PN v1.1.